spec/aws-record/record/item_operations_spec.rb

```diff
@@ -68,6 +68,31 @@ module Record
           )
         end
 
+        it 'passes through default api options' do
+          klass.configure_client(client: stub_client)
+          item = klass.new
+          item.id = 1
+          item.date = '2015-12-14'
+          item.body = 'Hello!'
+          # new record
+          item.save!(table_name: 'notused', return_values: 'ALL_OLD')
+          # forced
+          item.save!(force: true, table_name: 'notused', return_values: 'UPDATED_OLD')
+          # not updated tuple
+          item.save!(table_name: 'notused', return_values: 'ALL_NEW')
+          # updated tuple
+          item.clean!
+          item.body = 'Goodbye!'
+          item.save!(table_name: 'notused', return_values: 'UPDATED_NEW')
+
+          expect(api_requests).to include(
```

Comment:
I still feel like the expectations here are a little disconnected.  I'm fine with keeping this as one test case that does 4 different tests, but could we move the expectations closer to the tests?  

ie:
```
item.save!(table_name: 'notused', return_values: 'ALL_OLD')
expect(api_requests.last).to be(hash_including(table_name: 'TestTable', return_values: 'ALL_OLD'))
```

Reply:
It was awkward and redundant to do this .. I kept it as is but I found that I can use `match` to enforce ordering and strict api request size.